### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Next.js update

### CODE REVIEW

**Comment by davorpa** [comment]
> Next.js update

Are you sure? Why then is TOC fragments refactored and Bootstrap moved without move section items?

Please! Try to scope the PRs to it contents. E.g. all new resources joint in one and apart from listing structures

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -883,7 +885,6 @@
 #### Flask
 
 * [Flask Fridays](https://www.youtube.com/playlist?list=PLCC34OHNcOtolz2Vd9ZSeSXWc8Bq23yEz) - Codemy.com (YouTube)
-* [Flask Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTs4UjLw5MM6OjgkjFeUxCYH) - Corey Schafer
```
Comment (davorpa) [inline courses/free-courses-en.md:886]: Why remove this if PR is supposed to treat Next.js resources?

Comment (GH05T-HUNTER5) [reply courses/free-courses-en.md:886]: That's my mistake

**Comment by eshellman** [comment]
As @davorpa says, please try to scope your PRs to one issue at a time.

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Reverted by commit 04a439398daad674e1dc821bfa3fc5cac695b1ae

**Comment by davorpa** [comment]
> Are you sure? Why then is TOC fragments refactored and Bootstrap moved without move section items?

@eshellman too fast! After merge 271d67f4d993225e3049d753dd95db4538f7ad3a, the Bootstrap resources are out of sync with it place in TOC. Needs a hotfix PR to move them as `HTML and CSS` subsection

**Comment by eshellman** [comment]
> 

sorry about that. will hotfix if its still needed after reviewing all the other PRs.

**Comment by davorpa** [comment]
> > 
> 
> sorry about that. will hotfix if its still needed after reviewing all the other PRs.

Fixed by #7019

**Pull request merged**